Aptoide/aptoide-uploader
# [Feature/MOB-808] AutoLogin new design and Create Store bug fix

**What does this PR do?**

Creates a new AutoLogin view present before the existent Login view if the user is logged in vanilla.
Also fixes a bug where once the user got in the CreateStore view, he couldn't exit and opening the app again would jump to CreateStore view again

**Where should the reviewer start?**

- AutoLoginFragment.java
- AutologinPresenter.java
- AutoLoginManager.java

**How should this be manually tested?**

 - Logout from vanilla
 - Open Uploader and test that the new view doesnt show up
 - Go back to vanilla and Login
 - Check the new view and its information, and navigation to the other logins
 - Login with an account without store and try to exit the screen (a dialog should appear to confirm the user wants to go back)

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/MOB-808
  https://aptoide.atlassian.net/browse/MOB-862

**Code Review Checklist**
